internal/cert_test.go

```diff
@@ -1,3 +1,4 @@
+// Package internal contains private helper functions and models needed in client and server
```

Comment:
You only need the package comment once per package. You can put it in doc.go idiomatically if you have multiple files defining a package.

Reply:
Fixed thank you